google/cloud/bigtable/examples/table_admin_snippets.cc

```diff
@@ -292,7 +293,8 @@ void GetFamilyMetadata(google::cloud::bigtable::TableAdmin const& admin,
     std::cout << "Column family metadata for <" << family_name << "> is "
               << pos->second.DebugString() << "\n";
   }
-  // [END bigtable_get_family_metadata] [END bigtable_get_family]
+  // [END bigtable_get_family_metadata]
+  // [END bigtable_get_family]
```

Comment:
Nit: The non-nested ordering strikes my nested eye.

Reply:
@billyjacobson thoughts?

Reply:
good catch, fixed